collagraph/components/__init__.py

```diff
@@ -1,9 +1,25 @@
-from abc import ABCMeta, abstractmethod
+from abc import abstractmethod
 
 from observ import reactive, readonly
 
 
-class Component(metaclass=ABCMeta):
+RENDER_FUNCTION = None
+
+
+def set_render_function(fun):
+    global RENDER_FUNCTION
+    RENDER_FUNCTION = fun
+
+
+class Meta(type):
```

Comment:
you might want to rename this to `ComponentMeta` or something a little bit more specific